pkgs/development/node-packages/overrides.nix

```
@@ -444,6 +444,27 @@ final: prev: {
     '';
   };
 
+  readability-cli = prev.readability-cli.override (oldAttrs: {
+    # Wrap src to fix this build error:
+    # > readability-cli/readable.ts: unsupported interpreter directive "#!/usr/bin/env -S deno..."
+    #
+    # Need to wrap the source, instead of patching in patchPhase, because
+    # buildNodePackage only unpacks sources in the installPhase.
+    src = stdenv.mkDerivation {
+      src = oldAttrs.src;
+      name = oldAttrs.name + "-src";
+      patchPhase = "chmod a-x readable.ts";
+      installPhase = ''cp -r . "$out"'';
+    };
```

I think you can use `srcOnly`
```suggestion
    src = srcOnly {
      name = "readability";
      src = oldAttrs.src;
      patchPhase = "chmod -x readable.ts";
    };
```

---

Thank you. Good suggestion.